ansible/roles/qemu/files/kvm.conf

```diff
@@ -0,0 +1,3 @@
+options kvm_intel nested=1
```

Comment:
Reuse the existing file and ditto for all other places.

Reply:
Not sure what file you are talking about, but please feel free to modify and merge as you like.

Reply:
https://github.com/kholia/OSX-KVM/blob/master/kvm.conf